idaes/apps/grid_integration/coordinator.py

```diff
@@ -353,6 +356,64 @@ def _clone_tracking_model(self):
 
         return
 
+    def _update_staic_params(self, gen_dict):
+
+        """
+        Update static parameters in the Prescient generator parameter data dictionary.
+
+        Args:
+            gen_dict: Prescient generator parameter data dictionary.
+
+        Returns:
+            None
+        """
+
+        for param, value in self.bidder.bidding_model_object.model_data:
+            if param != "gen_name":
+                continue
+            elif param == "p_cost":
+                curve_value = convert_marginal_costs_to_actual_costs(value)
+                gen_dict[param] = {
+                    "data_type": "cost_curve",
+                    "cost_curve_type": "piecewise",
+                    "values": curve_value,
+                }
```

Comment:
Is this covered in an integration test? seems like important code to cover for Prescient interaction.

Reply:
Thanks @bknueven ! I fixed it!